### DISCUSSION TITLE
$fetch throws "body is disturbed" or "load error" when hitting server route.

### DISCUSSION THREAD

**Discussion by jawngee** [nuxt/nuxt#19527] [Questions]
This behavior just recently popped up, where using `$fetch` (or even `fetch`) when hitting a server api route throws either one of the above errors. This error is thrown BEFORE fetch does any network activity, so it's not the server route causing the problem.

Strangely, if I make the exact same request immediately after the error, then the request works as it should. So now I have to wrap all these fetch requests in retry blocks as a bandaid.

This also seems to be browser independent as the same issue shows up on Safari, Chrome, Firefox and Arc.

Has anyone ever run into this? Any tips on hunting this down?

**Comment by TheAlexLichter** [comment]
Can you create an issue with a reproduction?
